docker/configs/log4j.xml

```diff
@@ -27,19 +27,30 @@
 
     <appender name="console" class="org.apache.log4j.ConsoleAppender">
         <layout class="org.apache.log4j.PatternLayout">
-            <param name="ConversionPattern"
-                   value="%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p [%t] %c{1}:%L - %m%n"/>
+            <param name="ConversionPattern" value="%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p [%t] %c{1}:%L - %m%n"/>
+        </layout>
+    </appender>
+
+    <appender name="file" class="org.apache.log4j.RollingFileAppender">
+        <param name="append" value="false"/>
+        <param name="maxFileSize" value="100MB"/>
+        <param name="maxBackupIndex" value="3"/>
+        <param name="file" value="${APP_ROOT}/logs/bigben_app_${APP_PORT}.log"/>
```

Comment:
In production  docker environment, creating files with in docker environment can cause issues, but in this case, it's just max 300MB, so should be fine.

Reply:
The log directory is mapped to host system using LOG_DIR, so the files will be created outside of container. (see docker/app_run.sh)

Reply:
Sure not an issue. most of production environment will use Kubernetes or Mesos to orchestrate the containers. (or may use docker-compose, aws container management... ) 

certainly app_run.sh is good for development environment. I appreciate that.